### PR TITLE
Gird

### PR DESCRIPTION
**Pull request by VedanthB**
- added grid classes with media queries
- added grid template rows and columns classes
- added grid column / row span classes

https://user-images.githubusercontent.com/75097551/152438690-c56d64b6-6ff9-4fe4-861c-a95ba03a30d7.mov

### CODE REVIEW

**Review by SanjayJatti — APPROVED** [review]
Looking awesome!

File: components/grid.css [hunk 1]
```diff
@@ -0,0 +1,683 @@
+.container {
+  width: 100%;
+  max-width: 1200px;
```
Comment (shivanigangadharan) [inline components/grid.css:3]: Can use clamp here, for width like: `width: clamp(min_width, 100%, 1200px)`

**Review by shivanigangadharan — APPROVED** [review]
Looks good to merge. Nice work. 👍

File: components/grid.css (hunk 1 above)
Comment (VedanthB) [reply components/grid.css:3]: okay, thanks for pointing it out, i will use clamp here, didn't even think of it.

**Pull request merged**